packages/io/x3d-serializer/src/index.js

```diff
@@ -128,7 +130,7 @@ const convertObjects = (objects, options) => {
       shapes.push(convertPath2(object, options))
     }
   })
-  scene = scene.concat(shapes)
+  scene[1] = scene[1].concat(shapes)
```

Comment:
perhaps simplify to:
```js
const scene = [ 'Scene', [ 'Transform', { rotation: '1 0 0 -1.5708' }, shapes ] ]
```

Reply:
I kind of like the second option as that shows the hierarchy between scene and transform.

Reply:
will change.